Lib/asyncio/streams.py

```diff
@@ -303,6 +315,13 @@ def can_write_eof(self):
     def close(self):
         return self._transport.close()
 
+    def is_closing(self):
```

Comment:
Is this a new API method?  Please document it if it is.

Reply:
Forgot about the change - but exposing the method makes sense

Reply:
I think it's a bit too low-level. Can you imagine a good use case for async/await code that uses writer and reader?

Reply:
StreamWriter exposes all Transport methods except flow control related.
`is_closing()` is not about flow control, I see no reason to **not** implement it.

Reply:
OK, go ahead.